<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <title>Comparative Genomics of Clavibacter Bacteria - Danielle M. Stevens</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="icon" href="https://daniellemstevens.github.io/favicon.png">

  
  
  <link rel="stylesheet" href="/css/style.min.483c48f8e0c561eb041efb99630ec3665482a392192154701e0b451533ce13ed.css">
  

  

</head>

<body class='page page-default-single'>
  <div id="main-menu-mobile" class="main-menu-mobile">
  <ul>
    
    
    <li class="menu-item-home">
      <a href="/">
        <span>Home</span>
      </a>
    </li>
    
    <li class="menu-item-about">
      <a href="/about/">
        <span>About</span>
      </a>
    </li>
    
    <li class="menu-item-research">
      <a href="/research/">
        <span>Research</span>
      </a>
    </li>
    
    <li class="menu-item-cv">
      <a href="/cv/">
        <span>CV</span>
      </a>
    </li>
    
  </ul>
</div>
  <div class="wrapper">
    <div class='header'>
  <div class="container">
    <div class="logo">
      <img src = "IMG_0060.jpg"/>
      <a href = "https://github.com/DanielleMStevens"><img src = "GitHub-Mark-120px-plus.png" style = "width:30px;"/></a>
      <a href = "https://twitter.com/Dani_M_Stevens"><img src = "Twitter-featured.png" style = "width:40px;"/></a>
    </div>
    <div class="logo-mobile">
    </div>
    <div id="main-menu" class="main-menu">
  <ul>
    
    
    <li class="menu-item-home">
      <a href="/">
        <span>Home</span>
      </a>
    </li>
    
    <li class="menu-item-about">
      <a href="/about/">
        <span>About</span>
      </a>
    </li>
    
    <li class="menu-item-research">
      <a href="/research/">
        <span>Research</span>
      </a>
    </li>
    
    <li class="menu-item-cv">
      <a href="/cv/">
        <span>CV</span>
      </a>
    </li>
    
  </ul>
</div>
    <button id="toggle-main-menu-mobile" class="hamburger hamburger--slider" type="button">
  <span class="hamburger-box">
    <span class="hamburger-inner"></span>
  </span>
</button>
  </div>
</div>



    
    
    
    

    
    <div class="container pt-2 pt-md-6 pb-3 pb-md-6">
      <div class="row">
        
        <div class="col-12 col-md-3 mb-3">
          <div class="sidebar">
            
<div class="docs-menu">
  
  <ul>
    
    <li class="active ">
      <a href="https://daniellemstevens.github.io/research/comparative-genomics-of-clavibacter-bacteria/">Comparative Genomics of Clavibacter Bacteria</a>
    </li>
    
    <li class="">
      <a href="https://daniellemstevens.github.io/research/plant-immune-perception-of-gram-positive-bacteria/">Plant Immune Perception of Gram-positive Bacteria</a>
    </li>
    
    <li class="">
      <a href="https://daniellemstevens.github.io/research/plant-targets-of-tomato-pathogen-c.-michiganensis-protease-effectors/">Plant Targets of Tomato Pathogen C. michiganensis Protease Effectors</a>
    </li>
    
    <li class="">
      <a href="https://daniellemstevens.github.io/research/describing-the-variability-of-pamp-trigged-immune-reponse-in-citrus/">Describing the Variability of PAMP-triggered immune response in Citrus</a>
    </li>
    
  </ul>
</div>

          </div>
        </div>
        
        <div class="col-12 col-md-9">
          
<h1 class="title">Comparative Genomics of Clavibacter Bacteria</h1>
<div class="content ">
  <hr>
<br>
<h3 id="clavibacter-bacteria-are-host-specific-pathogens-with-many-predicted-virulence-effectors">Clavibacter bacteria are host specific pathogens with many predicted virulence effectors.</h3>
<p>Clavibacter species cause disease on a narrow host range of plants, but their ability to colonize endophytically is broad. There are six pathogens of the genus:</p>
<p>C. michiganensis (Cm - bacterial canker of tomato)
C. sepedonicus (Cs - potato ring rot)
C. capsici (Cc - bacterial canker of pepper)
C. insidiosus (Ci - bacterial wilt of alfalfa)
C. nebraskensis (Cn - Goss’s wilt of corn)
C. tessellarius (Ct - bacterial mosaic on wheat).</p>
<p>However, Cm and Cc break this rule, where both pathogens are weakly pathogenic on the related hosts, tomato and pepper. 10 The tomato pathogen Cm is the most-well studied. Cm carries multiple virulence genes on two plasmids, pCM1 and pCM2, and within a ~129Kb pathogenicity island (PAI), chp/tomA. While the plasmid number, size, and gene content vary within and across species, the entire chp/tomA PAI is only found in Cm. Instead, fragments and/or homologs of PAI members can be found in Cc, Cs, and Ci genomes. The chp/tomA PAI encodes suites of putative serine proteases of the Sbt, Chp, and Ppa families and carbohydrate activating enzymes (CAZymes). Functional validation of some of effectors include tomatinase, which degrades the plant defense compound α-tomatine, and celA, a cellulase which breaks down the cell wall component, cellulose. While this provides insight into mechanisms of disease for Cm in tomato, the plant targets of most effectors are unknown.4 Even less is known for other pathogens of the genus. While pathogen evolution is unclear, previous work suggests Clavibacter pathogens have undergone niche adaptation as a result of plant-microbe co-evolution.</p>
<p>To better assess weather effector mediate host range could be occuring, I have three main aims in my Ph.D.</p>
<h4 id="11-generate-a-robust-genomic-data-set-as-a-resource-for-clavibacter-research">1.1 Generate a robust genomic data set as a resource for Clavibacter research.</h4>
<p>In order to comprehensively predict effectors, I will generate a robust dataset of multiple high-quality genomes not well sampled in NCBI. I have collaborated with researchers to seqeunce diverse pathogenic Clavibacter  using Illumina and Oxford Nanopore technologies. By using both a long read and short read technology, we can leaverage on their individual strengths without drastically increasing costs. Long reads (nanopore), while more error prone, can provide scaffolding and span through repetative regions. The short reads (illumina), while very accruate for base calling, are prone to incomplete assemblies, particularly in regions which have repeats. So using nanopore and illumna can provide the best of both, long reads to provide a scaffold and short reads to correct for error. The completion of these genome sequences will provide a robust resource to investigate mechanisms governing pathogenicity and host range.</p>
<h4 id="12-predict-effectors-involved-in-pathogenicity-and-host-range">1.2 Predict effectors involved in pathogenicity and host range.</h4>
<p>Pathogen adaptation to a given environment has occured through gene gains, losses, and diversification. In my model, Clavibacter pathogens have adapted to a particular host through subtle changes in effector content, which can be predicted through a variety of computational approaches. As I&rsquo;ve learned from a wise postdoc, no one computational appraoch will give you the answer but a variety of robgust approaches will provide insight to help you move foreard and develop hypothesize to further test and validate. In total, I anticipate being able to predict effectors essential for pathogenicity and host range.</p>
<h4 id="13-functionally-assess-predicted-effector-loci">1.3 Functionally assess predicted effector loci.</h4>
<p>Functional assessment of in silico predictions will be performed using genetic and biochemical assays. Previous work supports proteases and CAZymes as candidates for mediating these processes. I anticipate analyzing the importance of ~5 groups of effectors and will tailor experiments based on their identity. The completion of these experiments will enable identification of the first host targets of Clavibacter effectors and their contribution to host range.</p>
<h5 id="publications-related-to-this-work">Publications related to this work:</h5>
<p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-phyto-082718-100124">Thapa, S. P. et al. The Evolution, Ecology, and Mechanisms of Infection by Gram-Positive, Plant-Associated Bacteria. Annu Rev Phytopathol 57, 341–365 (2019).</a></p>

</div>
</div>

        </div>
      </div>
    </div>
    
  </div>

  <div class="sub-footer">
  <div class="container">
    <div class="row">
      <div class="col-12">
        <div class="sub-footer-inner">
        </div>
      </div>
    </div>
  </div>
</div>

  

  
  

  
  <script type="text/javascript" src="/js/scripts.min.1237ff71925bb8625c97a9af8db4c54525258bedfd7c47493daaff723bea755e.js"></script>
  

  
  
  
    
  


</body>

</html>
